crates/torii/grpc/src/server/subscriptions/model_diff.rs

```diff
@@ -22,6 +22,8 @@ use super::error::SubscriptionError;
 use crate::proto;
 use crate::types::KeysClause;
 
+pub(crate) const LOG_TARGET: &str = "subscription";
```

Comment:
@glihm This subscription target is being used in multiple files. However I think its because its too generic. Let me know if I should updated in each file or move it to a common file as you mentioned in the previous review.

Reply:
imho we should add the module name as we may not have `model_diff` for instance inside each message. And this could ease the debugging to know which module of the subscriptions causes the error.